runtime/cudaq/builder/kernel_builder.h

```diff
@@ -204,6 +208,18 @@ void control(ImplicitLocOpBuilder &builder, std::string &name,
 void adjoint(ImplicitLocOpBuilder &builder, std::string &name,
              std::string &quakeCode, std::vector<QuakeValue> &values);
 
+void forLoop(ImplicitLocOpBuilder &builder, std::size_t start, std::size_t end,
```

Comment:
This is missing documentation. Same for the template.

Reply:
Fixed